AEPServices/Sources/ui/floating/FloatingButton.swift

```diff
@@ -63,6 +63,17 @@ public class FloatingButton: NSObject, FloatingButtonPresentable {
         }
     }
 
+    /// Set the Image for the floating button.
```

Comment:
My personal opinion here: But I don't think it is necessary to include doc comments on functions where the doc comment is defined in the protocol for the function.

Reply:
Agreed it was repetitive